Rules/Rules.csproj

```diff
@@ -67,4 +73,30 @@
     <DefineConstants>$(DefineConstants);PSV3;PSV4</DefineConstants>
   </PropertyGroup>
 
+  <PropertyGroup Condition="'$(TargetFramework)' == 'netcoreapp3.1' AND '$(Configuration)' == 'PSV7Release'">
+    <DefineConstants>$(DefineConstants);PSV7;CORECLR</DefineConstants>
+  </PropertyGroup>
+  <ItemGroup Condition="'$(TargetFramework)' == 'netcoreapp3.1' AND '$(Configuration)' == 'PSV7Release'">
+    <PackageReference Include="Microsoft.PowerShell.SDK" Version="7.0.0" />
```

Comment:
Why the SDK and not SMA?

Reply:
strictly speaking, SMA should have never been used, but the SDK did not exist. Using the SDK is the guidance we're providing.